addons/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/SerialMessage.java

```diff
@@ -154,7 +158,13 @@ public SerialMessage(int nodeId, byte[] buffer) {
         }
         this.priority = SerialMessagePriority.High;
         this.messageType = buffer[2] == 0x00 ? SerialMessageType.Request : SerialMessageType.Response;
-        this.messageClass = SerialMessageClass.getMessageClass(buffer[3] & 0xFF);
+        final int messageClassId = buffer[3] & 0xFF;
+        this.messageClass = SerialMessageClass.getMessageClass(messageClassId);
```

Comment:
I guess I'd question if we need both of these. One causes an exception, and one just a log entry - but they are really the same, other than we already added the 'unknown' label to the enum.

I would probably suggest not to add it to the enum (so remove the line 539 below) and just throw the exception. What do you think?